src/System.Private.Uri/src/System/IriHelper.cs

```diff
@@ -340,7 +340,7 @@ internal static unsafe string EscapeUnescapeIri(char* pInput, int start, int end
                     }
 
                     byte[] encodedBytes = new byte[maxNumberOfBytesEncoded];
-                    fixed (byte* pEncodedBytes = encodedBytes)
+                    fixed (byte* pEncodedBytes = &encodedBytes[0])
```

Comment:
Nit: the camel-casing of maxNumberOfBytesEncoded makes it harder to see that it's actually a const (and thus more easily validated as non-zero).  Could you fix the casing to conform to guidelines?